src/ngraph_builder.cc

```diff
@@ -95,8 +95,7 @@ std::shared_ptr<TOpType> ConstructNgNode(const std::string& op_name,
                                          TArg&&... Args) {
   auto ng_node = std::make_shared<TOpType>(std::forward<TArg>(Args)...);
   ng_node->set_friendly_name(op_name);
-  // Fails for now, so commenting out for now
-  // ng_node->add_provenance_tag(op_name);
+  ng_node->add_provenance_tag(op_name);
```

Comment:
Do we have enough tests (on other backends) to ensure that this won't crash?

Reply:
This line gets added in `ngraph_builder.cc` which means all the tests use this. gtests (opexecutor), pytests, tf python tests and resnet tests. Also in CI we run this on CPU and GPU 2 backends.

The issue was more related to running models with `pip install tensorflow` + `build ngraph bridge`
iIf TF is built from source then there is no problem. In fact the crash ws observed even with CPU backend in debian (where they had pip installed tensorflow). So its more a matter of gcc compatibility than backend